test/extended/util/cli.go

```diff
@@ -226,6 +258,17 @@ func (c *CLI) Args(args ...string) *CLI {
 	return c
 }
 
+// ExecWithoutExit executes a command in container ignoring non-zero exit value.
+// TODO: Remove this when https://github.com/kubernetes/kubernetes/pull/13728 is merged in OS
+func (c *CLI) ExecWithoutExit(pod, container, cmd string) *CLI {
```

Comment:
oh sorry, it is pending https://github.com/openshift/origin/pull/4611 I have to rework that.